### PR TITLE
show directory names in log output

### PR DESCRIPTION
**Pull request by abossard**
- added -simulate option, to show what would happen
- fixed but that occurred with weird characters in filenames (probably windows related)

### CODE REVIEW

**Comment by abossard** [comment]
Bugs, sorry. I'll fix it and add other small things as well.

**Comment by llakssz** [comment]
should i revert it?

**Comment by abossard** [comment]
No, its fixed now.

File: FunKiiU.py [hunk 1]
```diff
@@ -379,7 +387,11 @@ def main(titles=None, keys=None, onlinekeys=False, onlinetickets=False, download
                 continue
             elif title_id in titles:
                 continue
-            process_title_id(title_id, title_key, name, region, output_dir, retry_count, onlinetickets, patch_demo, patch_dlc)
+            process_title_id(title_id, title_key, name, region, output_dir, retry_count, onlinetickets, patch_demo, patch_dlc, simulate)
+
+
+def log(output):
```
Comment (abossard) [inline FunKiiU.py]: It seems that Python under Windows struggles to output some of the UTF-8 chars. It tried to convert it to CP850. This now ignores the errors during conversion. Which is fine for log output.

**Pull request merged**